### ISSUE TITLE
Rule Change: [max-lines] - add an option to ignore imports

### ISSUE THREAD

**Issue by avalanche1** [eslint/eslint#16195]
### What rule do you want to change?

max-lines

### What change to do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new option

### Example code

```js
```

### What does the rule currently do for this code?

https://github.com/eslint/eslint/issues/12875

### What will the rule do after it's changed?

https://github.com/eslint/eslint/issues/12875

### Participation

- [X] I am willing to submit a pull request to implement this change.

### Additional comments

_Pls see https://github.com/eslint/eslint/issues/12875. Counting imports in this rule is really giving Me headaches :pray::pray::pray:._

**Comment by nzakas** [comment]
Thanks for the input. As [stated in the README](https://github.com/eslint/eslint#stylistic-rule-updates), we actually aren't making any further updates to rules that enforce stylistic preferences. If you have a strong preference for such a change, you can always copy the existing rule into a custom rule and make it behave the way you want.
